contest/remote/vmksft-p.py

```diff
@@ -135,6 +171,22 @@ def _vm_thread(config, results_path, thr_id, in_queue, out_queue):
         out_queue.put({'prog': prog, 'test': test_name, 'file_name': file_name,
                        'result': result, 'time': (t2 - t1).seconds})
 
+        parse_tap = config.getboolean('ksft', 'parse_tap', fallback=False)
+        if parse_tap:
+            parse_nested_tests = config.getboolean('ksft', 'parse_nested_tests',
+                                                   fallback=False)
+            tests = _parse_tap(vm.log_out, parse_nested_tests)
+
+            for r_name, r_result in tests:
+                # skip the main test, already covered above
+                if r_name == f'selftests: {target}: {prog}':
+                    continue
+
+                out_queue.put({'prog': prog,
+                               'test': namify(r_name),
+                               'file_name': file_name,
+                               'result': r_result})
```

Comment:
I don't know if we should do that. We will still parse the "main" test (the selftest being tested) as before. So in case of a crash, we will still mark the selftest as failed. But I think we can mark the subtests that have ran successfully as "success", no? Maybe the crash was due to one particular subtest, and the rest is fine. No?